test/feature-benchmark/scenarios.py

```diff
@@ -803,3 +803,63 @@ class KafkaRecovery(KafkaScenario):
 true
 """
     )
+
+
+class KafkaRecoveryBig(Kafka):
+    SHARED = [
+        Td("$ kafka-create-topic topic=kafka-recovery-big partitions=8"),
+        # Ingest 10M records
+        Kgen(
+            topic="kafka-recovery-big",
+            args=[
+                "--keys=random",
+                "--num-records=10000000",
+                "--values=bytes",
+                "--max-message-size=32",
+                "--min-message-size=32",
+                "--key-min=256",
+                "--key-max=10000000",
+            ],
+        )
+        # Add 256 EOF markers
+        ,
+        Kgen(
+            topic="kafka-recovery-big",
+            args=[
+                "--keys=sequential",
+                "--num-records=256",
+                "--values=bytes",
+                "--min-message-size=2",
```

Comment:
can the value sizes stay the same for the eof records?

Reply:
Will do, np.